pkg/ca/fileca/fileca.go

```diff
@@ -103,8 +104,19 @@ func (fca *fileCA) CreateCertificate(_ context.Context, subject *challenges.Chal
 }
 
 func (fca *fileCA) Root(ctx context.Context) ([]byte, error) {
-	return pem.EncodeToMemory(&pem.Block{
-		Type:  "CERTIFICATE",
-		Bytes: fca.cert.Raw,
-	}), nil
+	fca.RLock()
+	defer fca.RUnlock()
+
+	buf := new(bytes.Buffer)
+	for _, cert := range fca.certs {
+		err := pem.Encode(buf, &pem.Block{
+			Type:  "CERTIFICATE",
+			Bytes: cert.Raw,
+		})
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return buf.Bytes(), nil
```

Comment:
add import for `github.com/sigstore/sigstore/pkg/cryptoutils` and then:

```suggestion
	return cryptoutils.MarshalCertificatesToPEM(fca.certs)
```

Reply:
An nice catch! Discovered the some of the test data didn't have the 100year expiration set and had expired so regenerated it.